pkg/utils/watchJob.go

```diff
@@ -83,3 +86,23 @@ func JobLogs(experimentName string, engineName string, client *kubernetes.Client
 	}
 	return 0, nil
 }
+
+//UpdateResultTable will update the result of pipelines in a table on github using python update script
+func UpdateResultTable(experimentName string, testVerdict string, engineName string, clientSet *chaosClient.LitmuschaosV1alpha1Client) (int, error) {
+
+	//Updating the result table
+	fmt.Println("The job_id for the job is:", os.Getenv("CI_JOB_ID"))
+	fmt.Println("The testVerdict for the experiment is:", testVerdict)
+	cmd := exec.Command("python3", "-u", "../utils/result_update.py", "--job_id", os.Getenv("CI_JOB_ID"), "--stage", "Experiment", "--test_desc", experimentName, "--test_result", testVerdict, "--time_stamp", (time.Now().Format(time.ANSIC)), "--token", os.Getenv("GITHUB_TOKEN"), "--test_name", experimentName)
+	var out bytes.Buffer
+	var stderr bytes.Buffer
+	cmd.Stdout = &out
+	cmd.Stderr = &stderr
+	err = cmd.Run()
+	if err != nil {
+		fmt.Println(fmt.Sprint(err) + ": " + stderr.String())
+		return 0, err
```

Comment:
Now i think return will require only err to return

Reply:
done @chandankumar4